src/main/java/finance/PnL.java

```diff
@@ -104,11 +103,13 @@ private void updatePnL() throws SQLException, IOException
                 {
                     String day = dayLookingAt.toString();
                     pnl = forwardPrice - Double.parseDouble(api.convert(currency, HOME_CURRENCY, day));
+                    pnl= action.equals("Buy") ? pnl : -pnl;
                 } else
                 {
                     String day = new Date(dayLookingAt.getTime()
                                           - (1000 * 60 * 60 * 24)).toString();
                     pnl = Double.parseDouble(api.convert(currency, HOME_CURRENCY, day));
+                    pnl= action.equals("Buy") ? pnl : -pnl;
```

Comment:
-You might want to swap this - if Sell -pnl otherwise +pnl 
-And take both versions outside of if/else since it looks the same
- And run ctrl + alt + l ;)